torch/_inductor/autotune_process.py

```diff
@@ -239,35 +256,14 @@ def benchmark_in_sub_process(
     """
     assert choice.bmreq is not None
     tuning_process.initialize()
-    assert tuning_process.valid()
-    process, request_queue, response_queue = (
-        tuning_process.process,
-        tuning_process.request_queue,
-        tuning_process.response_queue,
-    )
-    assert (
-        process is not None and request_queue is not None and response_queue is not None
-    )
-
-    request_queue.put(choice.bmreq)
-    while True:
-        try:
-            timing = response_queue.get(timeout=1.0)
-        except queue.Empty:
-            status = process.exitcode
-            if status is None:
-                # child process is still running
-                continue
-            # child process fail
-            assert status != 0
-
-            warnings.warn(
-                f"Fail to benchmark choice '{choice}'. It will be ignored. Please debug the root cause in case the choice can bring perf gains."  # noqa: B950 line too long
-            )
 
-            tuning_process.clear()
-
-            # return INF so this choice will be ignored
-            return float("inf")
-
-        return timing
+    tuning_process.put(choice.bmreq)
+    try:
+        return tuning_process.get()
+    except EOFError:
+        warnings.warn(
+            f"Failed to benchmark choice '{choice}'. It will be ignored. "
+            "Please debug the root cause in case the choice can bring perf gains."
```

Comment:
Please specify stack level of warnings.warn.

Reply:
Yeah, we need to fix this in the codebase to improve the warning levels: https://docs.python.org/3/library/warnings.html#warnings.warn the stack_level kwarg should be greater than the default because the default is basically useless (should be at least 2 so people can figure out where it's called from)